ArcadiaTeamsBot.sln

```diff
@@ -3,7 +3,12 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29806.167
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ArcadiaTeamsBot", "ArcadiaTeamsBot\ArcadiaTeamsBot.csproj", "{E32DF6FC-C173-4A35-9E2E-DAE6D1DB51D0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ArcadiaTeamsBot", "ArcadiaTeamsBot\ArcadiaTeamsBot.csproj", "{E32DF6FC-C173-4A35-9E2E-DAE6D1DB51D0}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{504D00B8-DC8D-42E4-826A-0F2AAB9863C1}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig

```

Comment:
Remove this and merge with develop, because @laska-sl have already added this file.

Reply:
Removed and merged